src/iop/basecurve.c

```diff
@@ -1532,6 +1550,45 @@ static void dt_iop_basecurve_sanity_check(dt_iop_module_t *self, GtkWidget *widg
   }
 }
 
+
+static gboolean _move_point_internal(dt_iop_module_t *self, GtkWidget *widget, float dx, float dy, guint state)
```

Comment:
So it looks like you just moved the function up?
Just add `static gboolean _move_point_internal(dt_iop_module_t *self, GtkWidget *widget, float dx, float dy, guint state);` here, and keep the function where it was.